### PR TITLE
Make Cardano transaction client stable

### LINKED ISSUE #1952: Make Cardano transactions stable in client
## Why
We want to release the Cardano transaction client in a stable version (it is currently only available as an unstable feature)

## What
Make the Cardano transaction client stable.

## How

- [x] Make the Cardano transaction client stable in:
  - [x] `mithril-client`
  - [x] `mithril-client-cli`
  - [x] `mithril-client-wasm`
  - [x] Example crates
  - [x] Documentation
  - [x] End-to-end tests
  - [x] Mithril Client multi-platform test manual workflow

### PR DESCRIPTION
**Pull request by dlachaume**
## Content

This PR includes the stabilization of the Cardano transaction client, previously available as an `unstable` feature.

This update ensures the Cardano transaction client is now stable using `mithril-client`, `mithril-client-cli` and `mithril-client-wasm`.

## Pre-submit checklist

- Branch
  - [ ] Tests are provided (if possible)
  - [ ] Crates versions are updated (if relevant)
  - [x] CHANGELOG file is updated (if relevant)
  - [x] Commit sequence broadly makes sense
  - [x] Key commits have useful messages
- PR
  - [x] No clippy warnings in the CI
  - [x] Self-reviewed the diff
  - [x] Useful pull request description
  - [x] Reviewer requested
- Documentation
  - [ ] Update README file (if relevant)
  - [x] Update documentation website (if relevant)
  - [ ] Add dev blog post (if relevant)

## Issue(s)

Closes #1952

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -27,6 +27,8 @@ As a minor extension, we have adopted a slightly different versioning convention
 
 - Support for `Cardano node` `9.2.1` in the signer and the aggregator.
 
+- Make the Cardano transaction client stable.
```
Comment (jpraynaud) [inline CHANGELOG.md:30]:
```suggestion
- Support for stable Cardano transaction client library, CLI and WASM.
```

**Review by jpraynaud — APPROVED** [review]
LGTM :+1:

**Review by Alenar — APPROVED** [review]
LGTM :rocket:

**Review by sfauvel — APPROVED** [review]
LGTM

**Pull request merged**